WordPressAuthenticator/Unified Auth/View Related/Password/PasswordViewController.swift

```diff
@@ -19,6 +19,12 @@ class PasswordViewController: LoginViewController {
 
     private let isMagicLinkShownAsSecondaryAction: Bool = WordPressAuthenticator.shared.configuration.isWPComMagicLinkShownAsSecondaryActionOnPasswordScreen
 
+    private let configurations = WordPressAuthenticator.shared.configuration
+
+    private var isSiteCredentialPassword: Bool {
+        loginFields.siteAddress.isEmpty == false && configurations.enableSiteCredentialsLoginForSelfHostedSites
```

Comment:
unrelated to this PR: might be nice having a screen mode enum since this screen is used for various use cases and it feels pretty fragile with conditional checks here and there 😅

Reply:
This is no longer used, so I've removed it in 7a7aebe541eb9beaa99e1505777b55e0ac9511d2.